packs/equipment/arctic-vigor-greater.json

```diff
@@ -9,7 +9,7 @@
         },
         "containerId": null,
         "description": {
-            "value": "<p>This tattoo takes the shape of the face of a roaring polar bear with piercing, ice-blue eyes. You don't take damage from extreme cold or severe cold. While exposed to incredible cold, you only take minor cold damage every hour, instead of moderate cold damage every minute. You ignore the uneven ground and difficult terrain caused by ice and the difficult terrain caused by snow (reducing greater difficult terrain from ice or snow to normal difficult terrain).</p>\n<hr />\n<p><strong>Activate</strong> <span class=\"action-glyph\">2</span> command</p>\n<p><strong>Frequency</strong> once per day</p>\n<hr />\n<p><strong>Effect</strong> You call forth a blast of polar wind in your choice of a @Template[burst|distance:10], @Template[burst|distance:15], or @Template[burst|distance:20] within a range of 60 feet that deals @Damage[12d6[cold]|traits:area-damage] damage. All creatures in the area must attempt a @Check[fortitude|dc:27|basic|name:Tattoo Polar Blast|traits:uncommon|showDC:all] save.</p>"
+            "value": "<p>This tattoo takes the shape of the face of a roaring polar bear with piercing, ice-blue eyes. You don't take damage from extreme cold or severe cold. While exposed to incredible cold, you only take minor cold damage every hour, instead of moderate cold damage every minute. You ignore the uneven ground and difficult terrain caused by ice and the difficult terrain caused by snow (reducing greater difficult terrain from ice or snow to normal difficult terrain).</p>\n<hr />\n<p><strong>Activate</strong> <span class=\"action-glyph\">2</span> command</p>\n<p><strong>Frequency</strong> once per day</p>\n<hr />\n<p><strong>Effect</strong> You call forth a blast of polar wind in your choice of a @Template[burst|distance:10], @Template[burst|distance:15], or @Template[burst|distance:20] within a range of 60 feet that deals @Damage[12d6[cold]|options:area-damage] damage. All creatures in the area must attempt a @Check[fortitude|dc:27|basic|name:Tattoo Polar Blast|traits:uncommon|showDC:all|options:area-effect] save.</p>"
```

Comment:
```suggestion
            "value": "<p>This tattoo takes the shape of the face of a roaring polar bear with piercing, ice-blue eyes. You don't take damage from extreme cold or severe cold. While exposed to incredible cold, you only take minor cold damage every hour, instead of moderate cold damage every minute. You ignore the uneven ground and difficult terrain caused by ice and the difficult terrain caused by snow (reducing greater difficult terrain from ice or snow to normal difficult terrain).</p>\n<hr />\n<p><strong>Activate</strong> <span class=\"action-glyph\">2</span> command</p>\n<p><strong>Frequency</strong> once per day</p>\n<hr />\n<p><strong>Effect</strong> You call forth a blast of polar wind in your choice of a @Template[burst|distance:10], @Template[burst|distance:15], or @Template[burst|distance:20] within a range of 60 feet that deals @Damage[12d6[cold]|options:area-damage] damage. All creatures in the area must attempt a @Check[fortitude|dc:27|basic|name:Tattoo Polar Blast|showDC:all|options:area-effect] save.</p>"
```